src/defaultSettings.js

```diff
@@ -16,52 +17,66 @@ const defaultSettings = {
             label: 'ALT',
             measure: 'Aminotransferase, alanine (ALT)',
             axis: 'x',
+            imputation: 'data-driven',
             cut: {
-                relative: 3,
+                relative_baseline: 3.8,
+                relative_uln: 3,
                 absolute: 1.0
             }
         },
         {
             label: 'ALP',
             measure: 'Alkaline phosphatase (ALP)',
+            axis: null,
+            imputation: 'data-driven',
             cut: {
-                relative: 1,
+                relative_baseline: 3.8,
+                relative_uln: 1,
                 absolute: 1.0
             }
         },
         {
             label: 'TB',
             measure: 'Total Bilirubin',
             axis: 'y',
+            imputation: 'data-driven',
             cut: {
-                relative: 2,
+                relative_baseline: 4.8,
+                relative_uln: 2,
                 absolute: 40
             }
         }
     ],
     missingValues: ['', 'NA', 'N/A'],
-    display: 'relative', //or "absolute"
+    axis_options: [
+        { label: 'Upper limit of normal adjusted (eDish)', value: 'relative_uln' },
+        { label: 'Baseline adjusted (mDish)', value: 'relative_baseline' },
+        { label: 'Raw Values', value: 'absolute' }
+    ],
+    display: 'relative_uln', //or "relative_baseline" or "absolute"
     baseline_visitn: '1',
     measureBounds: [0.01, 0.99],
     populationProfileURL: null,
     participantProfileURL: null,
+    point_size: 'Uniform',
+    point_opacity: false,
 
     //Standard webcharts settings
     x: {
         column: null, //set in onPreprocess/updateAxisSettings
         label: null, // set in onPreprocess/updateAxisSettings,
         type: 'linear',
         behavior: 'raw',
-        format: '.1f',
-        domain: [0, null]
+        format: '.2f'
+        //domain: [0, null]
     },
     y: {
         column: null, // set in onPreprocess/updateAxisSettings,
         label: null, // set in onPreprocess/updateAxisSettings,
         type: 'linear',
         behavior: 'raw',
-        format: '.1f',
-        domain: [0, null]
+        format: '.2f'
```

Comment:
worth adapting the precision to the variable?

Reply:
Worth doing for sure. I'll add the issue for a later release.